### PR TITLE
feat(bounties page): add editable gas input on bounty card

### PR DESCRIPTION
**Pull request by VERZUOL1**

### CODE REVIEW

File: services/sputnik/SputnikNearService/services/SputnikDaoService.ts [hunk 1]
```diff
@@ -143,22 +143,23 @@ export class SputnikDaoService {
         id,
         deadline,
       },
-      gas: GAS_VALUE,
+      gas: gas ? new BN(Number(gas) * 10 ** 15) : GAS_VALUE,
```
Comment (AlexeyPonomarenko) [inline services/sputnik/SputnikNearService/services/SputnikDaoService.ts]: I see that 10 ** 15 is being used in multiple places. Is there sense to move it to constant?

File: astro_2.0/components/TokenWidget/TokenWidget.tsx [hunk 2]
```diff
@@ -8,28 +8,30 @@ import styles from './TokenWidget.module.scss';
...
 export const TokenWidget: React.FC<TokenWidgetProps> = ({
   amount,
+  noIcon = false,
   icon,
   symbol,
   decimals,
 }) => {
   const isValid = useIsValidImage(icon);
 
   function renderIcon() {
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/TokenWidget/TokenWidget.tsx:25]: Maybe, we can make it in a following way
```
function renderIcon() {
    if (!noIcon) {
      if (symbol === 'NEAR') {
        return (
          <div className={styles.icon}>
            <Icon name="iconNear" width={24} />
          </div>
        );
      }

      if (isValid) {
        return (
          <div
            style={{
              backgroundImage: `url(${icon})`,
            }}
            className={styles.icon}
          />
        );
      }
    }

    return <div className={styles.icon} />;
  }
```

Comment (VERZUOL1) [reply astro_2.0/components/TokenWidget/TokenWidget.tsx:25]: fixed

File: services/sputnik/SputnikNearService/services/SputnikDaoService.ts (hunk 1 above)
Comment (VERZUOL1) [reply services/sputnik/SputnikNearService/services/SputnikDaoService.ts]: fixed

**Pull request merged**